apps/myaccount/src/components/federated-associations/federated-associations.tsx

```diff
@@ -297,22 +313,25 @@ export const FederatedAssociations: FunctionComponent<FederatedAssociationsProps
         );
     };
 
+    if (!showExternalLogins) {
+        return null;
+    }
+
     return (
-        showExternalLogins &&
-            (<SettingsSection
-                data-testid={ `${testId}-settings-section` }
-                description={ t("myAccount:sections.federatedAssociations.description") }
-                header={ t("myAccount:sections.federatedAssociations.heading") }
-                icon={ getSettingsSectionIcons().federatedAssociations }
-                iconMini={ getSettingsSectionIcons().federatedAssociationsMini }
-                iconSize="auto"
-                iconStyle="colored"
-                iconFloated="right"
-                showActionBar={ true }
-            >
-                { deleteConfirmation() }
-                { federatedAssociationsList() }
-            </SettingsSection>)
+        <SettingsSection
+            data-testid={ `${testId}-settings-section` }
+            description={ t("myAccount:sections.federatedAssociations.description") }
```

Comment:
better to add `data-componetid` as well

Reply:
Fixed with 479a0c10f8db49ce5b720bed4210e95e6037b122